### ISSUE TITLE
screen reader: Missing vocal feedback required on review stars

### ISSUE THREAD

**Issue by KateChuen** [SAP/spartacus#15039]
**Steps:**
Go to a pdp
Write a review
Focus on the review stars

**Result:**
Missing the vocal feedback "required".

![reviewStars](https://user-images.githubusercontent.com/45409515/154302485-0563badc-d2c0-4d59-a744-de1daddf8f44.png)

**Comment by developpeurweb** [comment]
A potential solution would look like this. (Example done manually by inserting attributes with inspector).

- Adding `tabindex` and `aria-label` to the group
- Adding `aria-label` and `role=button` to icons

![Screen Shot 2022-02-15 at 5 10 54 PM](https://user-images.githubusercontent.com/15113219/155436644-40cd9772-e0b3-47b8-afcc-1888997cc871.png)
![Screen Shot 2022-02-15 at 5 11 36 PM](https://user-images.githubusercontent.com/15113219/155436680-c603b3e5-ad4c-450a-8a99-c05abd19a2ff.png)
![Screen Shot 2022-02-15 at 5 12 47 PM](https://user-images.githubusercontent.com/15113219/155436717-6b55f3a8-911f-46a7-97b7-16380b503299.png)
![Screen Shot 2022-02-15 at 5 12 23 PM](https://user-images.githubusercontent.com/15113219/155436731-5a9bf657-4ef6-4584-8782-60cb0587f9d7.png)

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus using installation script
More extensive testing (on other screenreaders and devices) will be done with RC.

Ok now.
![Review3](https://user-images.githubusercontent.com/45409515/157904950-739dde43-3e8f-40ae-9e40-9e5c996f855d.png)
![Review2](https://user-images.githubusercontent.com/45409515/157904953-ea1a3d95-6eb1-444e-83d9-f72f09cae609.png)
![Review1](https://user-images.githubusercontent.com/45409515/157904955-433e353c-88dc-4733-ac03-aec843827f8f.png)
